### PR TITLE
sb_logger_util2 added

### PR DESCRIPTION
**Pull request by yravinderkumar33**

### CODE REVIEW

File: sb_logger_util2/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+var mkdirp = require('mkdirp');
+const fs = require('fs');
+const path = require('path');
+let loggerInit = false;
+let config;
+
+const init = (newConfig) => {
+
+    const defaultConfig = {
+        path: '../logs',
```
Comment (anoophm) [inline sb_logger_util2/index.js]: extension is missing

File: sb_logger_util2/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+const log4js = require('log4js');
+var mkdirp = require('mkdirp');
+const fs = require('fs');
+const path = require('path');
+let loggerInit = false;
```
Comment (anoophm) [inline sb_logger_util2/index.js]: change to loggerInitialized

File: sb_logger_util2/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+let config;
+
+const init = (newConfig) => {
```
Comment (anoophm) [inline sb_logger_util2/index.js]: change to appConfig

File: sb_logger_util2/index.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    });
+
+    loggerInit = true;
+}
+
+var processRequestObject = (request) => {
+    if (request) {
+        return {
+            id: request.id,
+            path: request.route.path,
```
Comment (anoophm) [inline sb_logger_util2/index.js]: just request.path

File: sb_logger_util2/package.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "sb_logger_util2",
```
Comment (anoophm) [inline sb_logger_util2/package.json]: sb_logger_util_v2

File: sb_logger_util_v2/index.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    });
+
+    loggerInitialized = true;
+}
+
+var processRequestObject = (request) => {
+    if (request) {
+        return {
+            id: request.id,
+            path: request.path,
```
Comment (anoophm) [inline sb_logger_util_v2/index.js]: we only need id remove rest

File: sb_logger_util_v2/index.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
+const log4js = require('log4js');
+var mkdirp = require('mkdirp');
```
Comment (HarishGangula) [inline sb_logger_util_v2/index.js]: const

**Review by HarishGangula — CHANGES_REQUESTED** [review]

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**